econml/tests/test_dml.py

```diff
@@ -1151,7 +1151,7 @@ def test_groups(self):
             est.fit(y, t, groups=groups)
 
         # test outer grouping
-        est = LinearDML(model_y=LinearRegression(), model_t=LinearRegression(), cv=GroupKFold(2))
```

Comment:
Is it worth adding some check to verify that a GroupKFold splitter was used under the hood?